docs/cmdline-opts/ssl-reqd.d

```diff
@@ -1,6 +1,6 @@
 Long: ssl-reqd
 Help: Require SSL/TLS
-Protocols: FTP IMAP POP3 SMTP
+Protocols: FTP IMAP POP3 SMTP LDAP (openldap only)
```

Comment:
I think the OpenLDAP limitation is better elaborated in the description, not in the protocols line. Maybe also then add that this is supported for LDAP starting in 7.81.0.